src/Pages/Products.js

```diff
@@ -0,0 +1,17 @@
+import {FilterDisplay} from '../components/Filters/filterDisplay'
+import {ProductsDisplay} from '../components/allProducts'
```

Comment:
Use Index.js in each folder to import all the components/pages from the folder.

Reply:
done